backend/migrations/0003_create_user_table.up.sql

```diff
@@ -0,0 +1,9 @@
+-- create_users_table.up
+CREATE TABLE IF NOT EXISTS user_profile(
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS user(
```
why not just `user`..? 😢

Reply:
[You don't listen to me ever do you ](https://www.postgresql.org/docs/current/sql-keywords-appendix.html)

:( 

I guess this kind of thing happens when a PR is a 3 book novel? I can't even find the comment I left about this, think its around page 175 in book 2

Reply:
@NeuroPyPy A "profile" is technically something else. Could we call this `fluke_user` instead then? Last thing